ForgeUnity/Assets/BeardedManStudios/Scripts/Multiplayer Menu/JoinMenu.cs

```diff
@@ -161,11 +180,9 @@ private void AddServer(string address, ushort port = NetWorker.DEFAULT_PORT, boo
 			var endpoint = HostResolver.Resolve(address, Settings.SQPPort);
 
 			serverListItemData.SqpQuery = sqpClient.GetQuery(endpoint);
+            serverListItemData.NextUpdate = Time.time + .2f;
```

Comment:
I'd still keep a random value in there so not all updates go out at the same time.

Reply:
I'll add it back